sample/src/main/kotlin/com/chuckerteam/chucker/sample/GraphQlTask.kt

```diff
@@ -6,23 +6,68 @@ import com.apollographql.apollo3.network.okHttpClient
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 import okhttp3.OkHttpClient
+import retrofit2.Callback
+import retrofit2.Call
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.create
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Query
 
+private const val GRAPHQL_BASE_URL = "https://rickandmortyapi.com/graphql/"
+private const val BASE_URL = "https://rickandmortyapi.com/"
 class GraphQlTask (
     client: OkHttpClient,
 ) : HttpTask {
 
     private val apolloClient = ApolloClient.Builder()
-        .serverUrl("https://rickandmortyapi.com/graphql")
+        .serverUrl(GRAPHQL_BASE_URL)
         .okHttpClient(client)
         .build()
 
+    private val api = Retrofit.Builder()
+        .baseUrl(BASE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
+        .client(client)
+        .build()
+        .create<Api>()
+
     private val scope = MainScope()
 
+    private val noopCallback = object: Callback<Any?> {
```

Comment:
This is unnecessary, you should be able to use two empty inline lambdas at the call site.